solutions/swb-reference/README.md

```diff
@@ -343,11 +842,34 @@ const { data: response } = await adminSession.resources.environments.get({status
 
 Go to `solutions/swb-app` to update `staticRouteConfig.ts` and `staticPermissionsConfig.ts` with any necessary changes to routes/permissions.
 
+## Reset User Password
+1. Go to the [Amazon Cognito console](https://console.aws.amazon.com/cognito/home) in your main account. If prompted, enter your AWS credentials.
+1. Choose **User Pools**.
+1. Choose your SWB user pool with name `swb-userpool-<STAGE>-<Region>`.
+1. Choose the **App integration** tab.
+1. Under **App client list** choose SWB app client with name `swb-client-<STAGE>-<Region>`.
+1. Under **Hosted UI** choose **View Hosted UI**.
+1. If the user has a temporary password, login with your user crendentials and follow isntructions to set a new password.
```

Comment:
nit: typo: follow the instructions to

Reply:
more clear to provide a link to this instructions?

Reply:
fixed typo